src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
```

Comment:
README should be professional and complete. A professional README should have the project and description. You may follow this README [template](https://github.com/microverseinc/readme-template) from Microverse. Moreover, the` App.js` component should be class-based.